spec/support/integration_helpers.rb

```diff
@@ -5,7 +5,7 @@ def sign_in(user, password = 'password')
     fill_in "Email",    with: user.email
     fill_in "Password", with: password
     click_button 'Sign in'
-    click_button 'Select Team'
+    find(:css, '.card-link', match: :first).click
```

Comment:
Integration tests should act like the user, and the user doesn't know the CSS class of a element. Don't we have a text for Capybara to click on?